tests/framework/session/test_session.py

```diff
@@ -380,6 +346,24 @@ def test_broken_config_loader(self, mock_settings_file_bad_config_loader_class):
         with pytest.raises(ValidationError, match=re.escape(pattern)):
             assert mock_settings.CONFIG_LOADER_CLASS
 
+    def test_no_logging_config(self, fake_project, caplog, mock_package_name, mocker):
+        caplog.set_level(logging.DEBUG, logger="kedro")
+
+        mocker.patch("subprocess.check_output")
+        session = KedroSession.create(mock_package_name, fake_project)
+        session.close()
+
+        expected_log_messages = [
+            "No project logging configuration loaded; "
+            "Kedro's default logging configuration will be used."
+        ]
+        actual_log_messages = [
+            rec.getMessage()
+            for rec in caplog.records
+            if rec.name == SESSION_LOGGER_NAME and rec.levelno == logging.DEBUG
+        ]
+        assert actual_log_messages == expected_log_messages
+
```

Comment:
This is the only really interesting new test.

Reply:
This test is specifically checking that we get that debug message, but the other tests aren't actually using logging config either right? So in theory they would all show that message in debug mode?

Reply:
Yes indeed! Though some of the other tests have `mocker.patch("kedro.framework.session.KedroSession._setup_logging")` so the debug message won't be emitted there anyway.